corehq/apps/reports/v2/models.py

```diff
@@ -3,11 +3,15 @@
 
 from collections import namedtuple
 
-from corehq.apps.reports.v2.exceptions import EndpointNotFoundError
+from corehq.apps.reports.v2.exceptions import (
+    EndpointNotFoundError,
+    ColumnFilterNotFoundError,
+)
 
 
 EndpointContext = namedtuple('EndpointContext', 'slug urlname')
-ColumnContext = namedtuple('ColumnContext', 'title slug width')
+ColumnMeta = namedtuple('ColumnMeta', 'title name width')
+FilterChoiceMeta = namedtuple('FilterOptionsMeta', 'name title')
```

Comment:
nit: the ordering of this is different to the `ColumnMeta` parameters. Can you make them the same for consistency?

Reply:
updated this in new PR https://github.com/dimagi/commcare-hq/pull/24099